src/containers/Accounts/AccountHeader/index.js

```diff
@@ -23,17 +23,26 @@ class AccountHeader extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      showBalanceSelector: false
+      showBalanceSelector: false,
+      prevProps: this.props
     };
     props.actions.loadAccountState(props.accountId);
   }
 
-  componentWillReceiveProps(nextProps) {
+  static getDerivedStateFromProps(nextProps, prevState) {
```

Comment:
I got rid of this function and the  Balance Selector seems to work now. Looks like it happened because prevState is always not equal to nextProps which always sets showBalanceSelector to false.